data/json/bionics.json

```diff
@@ -455,24 +739,41 @@
         "type": "bionic",
         "id": "bio_blaster",
         "name": "Fusion Blaster Arm",
+        "occupied_bodyparts": [
+            [ "ARM_L", 20 ],
+            [ "HAND_L", 5 ]
+        ],
         "act_cost": 50,
         "description": "Your left arm has been replaced by a heavy-duty fusion blaster!  You may use your energy banks to fire a damaging heat ray; however, you are unable to use or carry two-handed items, and may only fire handguns."
     },{
         "type": "bionic",
         "id": "bio_laser",
         "name": "Finger-Mounted Laser",
+        "occupied_bodyparts": [
+            [ "HAND_R", 1 ]
+        ],
         "act_cost": 50,
         "description": "One of your fingers has a small high-powered laser embedded in it.  This long range weapon is not incredibly damaging, but is very accurate, and has the potential to start fires."
     },{
         "type": "bionic",
         "id": "bio_emp",
         "name": "Directional EMP",
+        "occupied_bodyparts": [
+            [ "ARM_R", 1 ],
+            [ "HAND_R", 2 ]
```

Comment:
Would be cool if EMP could somehow prevent installing all "sensitive" electronics in the same hand (an EMP going off would fry them).